packages/framework/ini-files/config-specs.ini

```diff
@@ -1897,6 +1897,10 @@ opt-set-cmake-var ML_ENABLE_SuperLU BOOL FORCE : OFF
 use rhel7_sems-intel-2021.3-sems-openmpi-4.0.5_release-debug_shared_no-kokkos-arch_no-asan_no-complex_fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables
 use PACKAGE-ENABLES|ALL
 
+opt-set-cmake-var Zoltan_ENABLE_Scotch BOOL FORCE                       : OFF
```

Comment:
Do you really want to run off all Scotch support in Zoltan?

Reply:
@jmlapre: I know you're working on this already but please disable 3 failing zoltan tests with the:
```TEST_NAME_DISABLE=ON``` syntax.